src/utils/createGenerateClassName.js

```diff
@@ -15,5 +21,23 @@ const jssCounter = globalRef[namespace]++
 export default (): generateClassName => {
   let ruleCounter = 0
 
-  return (rule: Rule): string => `${rule.key}-${jssCounter}-${ruleCounter++}`
+  return (rule: Rule, sheet?: StyleSheet): string => {
+    ruleCounter += 1
+
+    if (ruleCounter > maxRules) {
+      warning(
+        false,
+        'You might have a memory leak. Rule counter is at %s.',
+        ruleCounter
+      )
+    }
+
+    if (env === 'production') {
+      return `c${jssCounter}${ruleCounter}`
```

Comment:
Please make this configureable. For some legacy reasons, I have some global styles that was created in this same manner. I'm using a fork of Material UI to avoid this right now and would love to have an option that I can easily configure instead of a fork